servers.json

```diff
@@ -20,6 +20,13 @@
             "jailmc.com"
         ]
     },
+    {
+        "id": "neptuniummc",
+        "name": "NeptuniumMC Italia",
+        "addresses": [
+            "play.neptuniummc.ml"
```

Comment:
Please remove the `play.` prefix to the domain;

The system automatically detects sub-domains, [click here](https://github.com/LunarClient/ServerMappings/blob/master/README.md#ip-addresses) for more information on ip-addresses.

Reply:
Thanks!